lux/core/frame.py

```diff
@@ -498,9 +498,15 @@ def maintain_recs(self, is_series="DataFrame"):
                 if len(vlist) > 0:
                     rec_df._recommendation[action_type] = vlist
             rec_df._rec_info = rec_infolist
+            quantitative_columns = [i for i in self.dtypes if i != "O" and i != "str"]
```

Comment:
dtypes is pandas dtypes right? So what if the columns are of type DateTime, object, bools, etc? maybe make this only include i if "int" or "float" is in the type.

Reply:
i believe datetime works as a quantitative attr and there's lots of other types besides "int" and "float" bc bytes are specified (i think 8 total) so i figured it was easier this way